tfexec/validate_types.go

```diff
@@ -0,0 +1,30 @@
+package tfexec
+
+// TODO: move these types to terraform-json
```

Comment:
Is this project 1.0? Would it be a breaking change to export these types, and then move them to tfjson and remove them here?

Reply:
I have just raised https://github.com/hashicorp/terraform-json/pull/23 - will see if I can get it merged 🔜